sim/paladin/item_sets_pve.go

```diff
@@ -187,3 +187,90 @@ var ItemSetMercifulJudgement = core.NewItemSet(core.ItemSet{
 		},
 	},
 })
+
+var ItemSetRadiantJudgement = core.NewItemSet(core.ItemSet{
+	Name: "Radiant Judgement",
+	Bonuses: map[int32]core.ApplyEffect{
+		2: func(agent core.Agent) {
+			// 2 pieces: Increases damage done by your damaging Judgements by 20% and your Judgements no longer consume your Seals on the target.
+			paladin := agent.(PaladinAgent).GetPaladin()
+			core.MakePermanent(paladin.RegisterAura(core.Aura{
```

Comment:
I learned that you can omit a Duration when using just an `OnInit` dummy aura so you could remove `MakePermanent` here for a small performance gain since it won't activate on each reset

Reply:
changed